docs/MS Excel/Формулы/Округление.md

```diff
@@ -0,0 +1,87 @@
+
+### ОКРУГЛ
+
+Функция ОКРУГЛ округляет число до ближайшего целого числа. Например, если у вас есть число 3.14159, функция ОКРУГЛ округлит его до 3, а если у вас есть число 3.7, функция ОКРУГЛ округлит его до 4.
+
+### ОКРУГЛВВЕРХ
+
+Функция ОКРУГЛВВЕРХ округляет число до целого числа в большую сторону. Например, если у вас есть число 3.14159, функция ОКРУГЛВВЕРХ округлит его до 4, а если у вас есть число -2.5, функция ОКРУГЛВВЕРХ округлит его до -2.
+
+### ОКРУГЛВНИЗ
+
+Функция ОКРУГЛВНИЗ округляет число до целого числа в меньшую сторону. Например, если у вас есть число 3.14159, функция ОКРУГЛВНИЗ округлит его до 3, а если у вас есть число -2.5, функция ОКРУГЛВНИЗ округлит его до -3.
+
+### Примеры использования функций округления:
+
+- Пример округления числа с помощью функции ОКРУГЛ:
+
+```
+ОКРУГЛ(3.14159) = 3
+ОКРУГЛ(3.7) = 4
+```
+
+- Пример округления числа в большую сторону с помощью функции ОКРУГЛВВЕРХ:
+
+```
+ОКРУГЛВВЕРХ(3.14159) = 4
+ОКРУГЛВВЕРХ(-2.5) = -2
+```
+
+- Пример округления числа в меньшую сторону с помощью функции ОКРУГЛВНИЗ:
+
+```
+ОКРУГЛВНИЗ(3.14159) = 3
+ОКРУГЛВНИЗ(-2.5) = -3
+```
+
+В Microsoft Excel эти функции могут быть использованы для автоматической обработки данных и получения точных результатов. Например, при работе с процентами, вам может понадобиться округлить число до сотых или процентов. В таких случаях функция ОКРУГЛВНИЗ будет полезна для округления до меньшей стороны, а функция ОКРУГЛВВЕРХ - для округления до большей стороны.
+
+Кроме того, в Excel существует функция ROUND, которая позволяет округлить число до указанного числа десятичных знаков. Например, если вы хотите округлить число до двух десятичных знаков, вы можете использовать формулу:
+
+`=ROUND(число, 2)`
+
+Также в Excel есть функция MROUND, которая позволяет округлить число до указанного целого числа, указывая положительное или отрицательное число для округления в большую или меньшую сторону. Например, если вы хотите округлить число к ближайшему целому числу, вы можете использовать формулу:
+
+`=MROUND(число, 1)`
+
+Кнопки увеличения и уменьшения разрядности на калькуляторах и компьютерах позволяют быстро изменять точность чисел с десятичной запятой. Использование этих функций и методов позволяет работать с числами более эффективно и точно.
+
+В Excel есть еще несколько функций, которые могут быть полезными для округления чисел:
+
+1. CEILING - округление числа в большую сторону без остатка. Формула: `=CEILING(число, значение_шага)`
+2. FLOOR - округление числа в меньшую сторону без остатка. Формула: `=FLOOR(число, значение_шага)`
+3. EVEN - округление числа в меньшую сторону, если оно нечетное, и в большую сторону, если оно четное. Формула: `=EVEN(число)`
+4. ODD - округление числа в меньшую сторону, если оно четное, и в большую сторону, если оно нечетное. Формула: `=ODD(число)`
+
+Эти функции могут быть особенно полезны при работе с данными, которые требуют точной обработки и контроля над округлением чисел.
+
+
```

Comment:
[ch-md-error-separate-blocks](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-separate-blocks)